xml/System.Collections.Generic/Dictionary`2.xml

```diff
@@ -3410,11 +3411,11 @@ Unlike the <xref:System.Collections.Generic.Dictionary%602.Add%2A> method, this
         </Parameter>
       </Parameters>
       <Docs>
-        <typeparam name="TAlternateKey">To be added.</typeparam>
-        <param name="lookup">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TAlternateKey">The alternate type of a key for performing lookups.</typeparam>
+        <param name="lookup">The created lookup instance when the method returns true, or a default instance that should not be used if the method returns false.</param>
+        <summary>Gets an instance of a type that may be used to perform operations on the current <see cref="T:System.Collections.Generic.Dictionary`2"/> using a <typeparamref name="TAlternateKey"/> as a key instead of a <typeparamref name="TKey"/>.</summary>
```

Comment:
```suggestion
        <summary>Gets an instance of a type that can be used to perform operations on the current <see cref="T:System.Collections.Generic.Dictionary`2"/> using a <typeparamref name="TAlternateKey"/> as a key instead of a <typeparamref name="TKey"/>.</summary>
```